geo/src/algorithm/centroid.rs

```diff
@@ -63,6 +63,24 @@ where
 {
     type Output = Point<T>;
 
+    /// The Centroid of a [`Line`] is it's middle point
```

Comment:
Should be "its"!

Reply:
Fixing this up everywhere in the file, had multiple occurences